docs/cluster-configuration.md

```diff
@@ -206,12 +206,6 @@ VMI example:
 
 **Default**: `false`
 
-### enableManagedTenantQuota Feature Gate
-If set to true, enables the Managed Tenant Quota (MTQ) feature. See more details 
-[here](https://github.com/kubevirt/managed-tenant-quota).
-
-**Default**: `false`
-
```

Comment:
Please don't delete, but just say it's deprecated and ignored

Reply:
Done